community/bolt/src/docs/dev/serialization.asciidoc

```diff
@@ -407,8 +408,9 @@ Relationship (signature=0x52) {
 
 [[bolt-value-pathstruct]]
 ==== Path
-A Path represents a walk through a graph.
-Its representation consists of a set of distinct nodes included within the path, a similar set of distinct relationships and a sequence describing the shape of the path.
+A Path is a sequence of alternating nodes and relationships corresponding to a walk in the graph.
+The sequence always begins and ends with a node.
+Its representation consists of a list of distinct nodes, a list of distinct relationships and a sequence of integer indexes describing the shape of the path.
```

Comment:
I would suggest "integer indexes" -> "integers". 
Also, "describing the shape of the path" -> "describing the path traversal"